src/lib/Page.svelte

```diff
@@ -9,6 +9,11 @@
   import Chart from './Chart.svelte';
   import BasicBarchart from './BasicBarchart.svelte';
   import StackedBarchar from './StackedBarchart.svelte';
+  import { onMount } from 'svelte';
+  import * as d3 from 'd3';
+  onMount(() => {
+    d3.selectAll('a').attr('target', '_blank');
+  });
```

Comment:
Hey fyi, I don't think we want this going in for everyone. We don't really want to load d3 into every project if there's no need, and there are lots of reasons you may want to _not_ make a link external, so doing this across the board isn't the answer.

Reply:
Sounds good. How about keeping these lines as a comment so that people can simply uncomment these lines if they do want to open new links in new tabs?